boards/arm/v2m_musca/doc/index.rst

```diff
@@ -374,7 +374,7 @@ for the secure veneers.
 
 .. code-block:: bash
 
-   cp -r install/ $ZEPHYR_PROJECT/src/ext
```

Comment:
Hi

Just curious was it led some function errors when having the slash, or just make the alignment?

This change is fine to me, thanks for figure this out.

Please be aware that we are going to include TFM repo and compile TFM automatically when building Zephyr. https://github.com/zephyrproject-rtos/zephyr/pull/19985

Reply:
Hi @karl-zh , the problem is that with the extra '/' the path to the veneers object on the subsequent line is off by one path segment.  Not a big issue, one can easily figure that out :-)

In all cases, thanks for the pointer to #19985.  I see some excellent stuff there, which is going to save a lot of time and silly mistakes.  Thanks!

Reply:
I see, thank you to have this fixed. Please give more description in your commit message, and follow what @dbkinder mentioned.